lang_python.adoc

```diff
@@ -13,130 +13,155 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python [--python-package .] my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use Python 3
+because Python 2 is EOL since 2020-01-01.
 
-TODO: document `--python-package`
+`--python-package` is an optional CLI arg allows you to select a parent package
+name used by one generated file to import symbols from another one.
 
-[[add-runtime]]
-=== Adding runtime library
+It is recommended to just follow the algorithm:
+
+* If you don't want to get an `ImportError`, determine if you need `--python-package`:
```

Comment:
This list item (which brings an additional level of nesting) is IMHO completely unnecessary at all. Who would want to get an `ImportError`? Everybody wants everything to work seamlessly on the first try without errors. Rather skip directly to the point. You can marginally mention at the end of the sentence somewhere that when you don't do this, you'll receive an `ImportError`, but it's not the main point.

Reply:
Removed.